index.bs

```diff
@@ -776,19 +789,19 @@ interface XRReferenceSpace : XRSpace {
 };
 </pre>
 
-Each {{XRReferenceSpace}} has a <dfn for="XRReferenceSpace">type</dfn>, which is an {{XRReferenceSpaceType}}.
+The [=XRSpace/internal type=] of an {{XRReferenceSpace}} is one of the variants of {{XRReferenceSpaceType}}, chosen during construction..
 
 An {{XRReferenceSpace}} is most frequently obtained by calling {{XRSession/requestReferenceSpace()}}, which creates an instance of an {{XRReferenceSpace}} or an interface extending it, determined by the {{XRReferenceSpaceType}} enum value passed into the call. The type indicates the tracking behavior that the reference space will exhibit:
 
-  - Passing a type of <dfn enum-value for="XRReferenceSpaceType">identity</dfn> creates an {{XRReferenceSpace}} instance. It represents a reference space where the [=viewer=] is always at the [=native origin=]. Only the {{XRSession/viewerSpace}} and {{XRInputSource/targetRaySpace}}'s of {{XRInputSource}}'s with an {{XRInputSource/targetRayMode}} of {{XRTargetRayMode/screen}} can be spatially related to an {{identity}} reference space.
+  - Passing a type of <dfn enum-value for="XRReferenceSpaceType">identity</dfn> creates an {{XRReferenceSpace}} instance. Its [=native origin=] is the position and orientation of the [=viewer=]. Only the {{XRSession/viewerSpace}} and {{XRInputSource/targetRaySpace}}'s of {{XRInputSource}}'s with an {{XRInputSource/targetRayMode}} of {{XRTargetRayMode/screen}} can be spatially related to an {{identity}} reference space.
 
-  - Passing a type of <dfn enum-value for="XRReferenceSpaceType">position-disabled</dfn> creates an {{XRReferenceSpace}} instance. It represents a tracking space where orientation is tracked but the [=viewer=]s position is always at the [=native origin=].
+  - Passing a type of <dfn enum-value for="XRReferenceSpaceType">position-disabled</dfn> creates an {{XRReferenceSpace}} instance. Its [=native origin=] has a position that always remains equal to the position of the [=viewer=], and a constant orientation initialized based on the conventions of the underlying [=/XR device=]. It can be used to create experiences where the movement of the user is disregarded, as it will not report changes in position when used with {{XRFrame/getViewerPose()}}.
```

Comment:
Now that we're settling in on clean physically-based reference space definitions (which is great, especially for ensuring the associativity of space relationships!), I wonder if "position-disabled" reference spaces will operate as we intend for their 360-degree video scenarios when on stereo headsets.

Under this definition, `getViewerPose` would still return non-identity eye positions for each `XRView` relative to this reference space, since the eyes are indeed in physically different places.  That would cause the app to render each eye from some pose other than (0, 0, 0), leading to weird parallax artifacts relative to its finite-sized video sphere, which is the primary problem that "position-disabled" was trying to avoid.  I'd really hate to see us backpedal on the physically-based definitions and introduce some special case to `getViewerPose` for "position-disabled" spaces to magically zero out the `XRView` poses, since we should otherwise be fully clean on reference spaces behaving identically, just with differently-behaving origins.

Alternatively, **what if we just replaced the "position-disabled" reference space with a `disablePosition` option on `getViewerPose`**?  360-video apps could then just use the "eye-level" reference space like other seated experiences, with their one difference being a parameter they pass to one function whose behavior they actually wish to impact.  By adding the option to the affected method, all other use of reference spaces stays clean and easy to understand.

The simplest option here would just be to require that 360-video apps use a big enough video sphere (e.g. 1000m) that these parallax issues from neck modeling and stereo separation in "eye-level" spaces would fall away.  If we do that, **we could just cut the "position-disabled" space entirely**.  Perhaps in the interest of getting quickly to WebXR 1.0, that's the best answer?

Reply:
Filed #625 to track this issue, which is broader than this particular change.